### PR TITLE
some nice fresh loading messages

### PR DESCRIPTION
**Pull request by brine**

### CODE REVIEW

File: octgnFX/Octgn/Resources/StartupMessages.txt [hunk 1]
```diff
@@ -7,73 +7,63 @@ Counting backwards from infinity...
...
+Increasing processing speed to 88 miles per hour...
 The internet is currently at capacity. Please wait for the next available opening.
 Installing the enchanted bunny...
-Please wait while this nice invisible progress bar slowly fills to 100%.
-Sorry for the inconvenience, but I still have some more things to load.
+Please wait while this nice invisible progress bar slowly fills to 100%...
 Gah, go read a book, I'm nearly finished!
-Oh, no! Loading time...
-Please wait, the bits are still breeding.
+Oh, no! Motr loading screens...
```
Comment (kellyelton) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: WTF is Motr?

File: octgnFX/Octgn/Resources/StartupMessages.txt [hunk 2]
```diff
@@ -83,7 +73,26 @@ Shouldn't you be sleeping?
...
+Please wait, uninstalling our competitors' software...
+Oadinglay Easeplay Aitway
+tiaw esaelp gnidaol
+Checking for updates that will probably break OCTGN...
+I don't even get paid to write these!
+Level up! Loading Speed +3
+Admit it, you enjoy loading screens...
+Vacuuming the game lobby...
+Randomly swapping your card images...
+Installing an even worse card shuffler...
```
Comment (kellyelton) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: ....srs?

File: octgnFX/Octgn/Resources/StartupMessages.txt [hunk 3]
```diff
@@ -83,7 +73,26 @@ Shouldn't you be sleeping?
...
+Calculating a reasonable loading time...
+Making OCTGN great again, please wait...
+INSERT LOADING MESSAGE HERE
+ERROR 533b: Loading Message Not Found
+Surrendering to the robotic overlords...
+Loading a slightly worse alternate universe...
+Please wait, uninstalling our competitors' software...
+Oadinglay Easeplay Aitway
+tiaw esaelp gnidaol
+Checking for updates that will probably break OCTGN...
```
Comment (kellyelton) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: Maaan

File: octgnFX/Octgn/Resources/StartupMessages.txt [hunk 4]
```diff
@@ -83,7 +73,26 @@ Shouldn't you be sleeping?
 What are you staring at?
 You're my best friend.  Love, OCTGN <3
 For loading messages in English, please press 1.
-Loading way more data than necessary...
+Loading waaaaaay more data than necessary...
+Calculating a reasonable loading time...
+Making OCTGN great again, please wait...
```
Comment (kellyelton) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: Make us look like weiners

File: octgnFX/Octgn/Resources/StartupMessages.txt (hunk 1 above)
Comment (brine) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: oops

Comment (kellyelton) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: I love it, lets leave it in.

Comment (kellyelton) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: I Found out what a Motr is...

Comment (kellyelton) [inline octgnFX/Octgn/Resources/StartupMessages.txt]: So I actually kind of agree with the sentiment "Oh, no! Motr loading screens..." hahahaha

**Pull request merged**